packages/netlify-cms-backend-github/src/API.ts

```diff
@@ -655,7 +651,9 @@ export default class API {
   }
 
   async fetchBlobContent({ sha, repoURL, parseText }: BlobArgs) {
-    const result: Octokit.GitGetBlobResponse = await this.request(`${repoURL}/git/blobs/${sha}`);
+    const result: Octokit.GitGetBlobResponse = await this.request(`${repoURL}/git/blobs/${sha}`, {
```

Comment:
Since `sha` is a part of the URL and unique per blob we can safely always get it from the cache.